water_use_gifs/R/draw_utils.R

```diff
@@ -0,0 +1,113 @@
+
+
+make_arc <- function(x0, y0, r, from_angle, to_angle){
+  theta <- seq(from_angle, to_angle, by = 0.002)
+  x_out <- rep(NA, length.out = length(theta))
+  y_out <- rep(NA, length.out = length(theta))
+  for (i in 1:length(theta)){
```

Comment:
I'm quite tolerant of loops in R relative to a lot of people, but this is the kind that should be avoided (even with the pre-allocation of x_out and y_out). `x_out = x0 + r*cos(theta)` works:
```
system.time(for (i in 1:length(theta)){x_out2[i] = x0 + r*cos(theta[i])})
   user  system elapsed 
   0.02    0.00    0.01 
system.time({x_out = x0 + r*cos(theta)})
   user  system elapsed 
      0       0       0
all.equal(x_out, x_out2)
[1] TRUE
```

Reply:
Thanks! I think I forgot how to R :( 
fixed